suite-native/assets/src/assetsSelectors.ts

```diff
@@ -80,3 +80,9 @@ export const selectAssetsWithBalances = memoize((state: AssetsRootState) => {
         })
         .filter(data => data !== undefined) as AssetType[];
 });
+
+export const selectIsAccountsPortfolioEmpty = (state: AssetsRootState) => {
```

Comment:
```suggestion
export const selectIsPortfolioAccountsEmpty = (state: AssetsRootState) => {
```

Reply:
The plural is fine with me. The order of the words matters as the accounts are within portfolio not the other way around. But maybe just simplify it and use `selectIsPortfolioEmpty` or `selectHasDashboardAccounts` or similar.

Reply:
https://github.com/trezor/trezor-suite/commit/019dcf358b6a327c6c54804d511837ec2b74aa82 renamed to `selectIsPortfolioEmpty`